src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -175,29 +173,13 @@ static Config merge_configs(const Config& globalConfig,
     return localConfig;
 }
 
-static auto get_specified_device_name(const Config config) {
-    if (config.has<DEVICE_ID>()) {
-        return config.get<DEVICE_ID>();
-    }
-    return std::string();
-}
-
-static Config add_platform_to_the_config(Config config, const std::string_view platform) {
-    config.update({{ov::intel_npu::platform.name(), std::string(platform)}});
-    return config;
-}
-
 Plugin::Plugin()
     : _options(std::make_shared<OptionsDesc>()),
       _globalConfig(_options),
       _logger("NPUPlugin", Logger::global().level()) {
     OV_ITT_SCOPED_TASK(itt::domains::NPUPlugin, "Plugin::Plugin");
     set_device_name("NPU");
 
-    registerCommonOptions(*_options);
-    registerCompilerOptions(*_options);
-    registerRunTimeOptions(*_options);
-
     // parse env_variables to get LOG_LEVEL if needed
     _globalConfig.parseEnvVars();
```

Comment:
I assume the config is empty here because no option registered now, shall we remove it? And env check is put off to init_option(), the plugin log from here to the function can not show with OV_NPU_LOG_LEVEL change. If we have env in option to change backend behavior, I assume it also not work with current change.

Reply:
Correct. This is a usecase which slipped my attention, need to check.